test/elvis_SUITE.erl

```diff
@@ -187,7 +192,7 @@ main_git_hook_fail(_Config) ->
 
         meck:new(elvis_git, [passthrough]),
         LongLine = <<"Loooooooooooooooooooooooooooooooooooooooooooooooooooong ",
-                     "Liiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiine">>,
+                     "Liiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiine">>,
```

Comment:
Line 195 is too long:                      "Liiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiine">>,.